barretenberg/acir_tests/gen_inner_proof_inputs.sh

```diff
@@ -28,18 +28,18 @@ VFLAG=${VERBOSE:+-v}
 RFLAG=${RECURSIVE:+-r}
 
 echo "Write VK to file for assert_statement..."
-$BIN write_vk $VFLAG -c $CRS_PATH -o
+$BIN write_vk $VFLAG -c $CRS_PATH -o ./target/vk
```

Comment:
added a bunch of options because main.ts didn't set default options. This script did not work for the wasm flow before this